features/facebookImport/src/model/importer.js

```diff
@@ -1,55 +1,159 @@
-import OffFacebookEventsImporter from "./importers/off-facebook-events-importer.js";
-import AdInterestsImporter from "./importers/ad-interests-importer.js";
-import ConnectedAdvertisersImporter from "./importers/connected-advertisers-importer.js";
-import ConnectedAdvertisersAllTypesImporter from "./importers/connected-advertisers-all-types-importer.js";
-import InteractedWithAdvertisersImporter from "./importers/interacted-with-advertisers-importer.js";
-import FriendsImporter from "./importers/friends-importer.js";
-import FollowedPagesImporter from "./importers/pages-followed-importer.js";
+import OffFacebookEventsImporter, {
```

Comment:
Yeah could be a static. These changes take sooo long 😢

Reply:
Added them to the class

Reply:
I think we later on we can simply use the class name instead of a separate key (and save the results into a dedicated object inside `DataAccount`, just how the analyses do it), but for now I think it's good to keep them around.